src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -238,19 +239,24 @@
         [rn/view {:margin-top 12}]
         [quo/community-tags tags]
         [preview-user-list]
-        (when-not joined
-          [quo/button
-           {:on-press  #(rf/dispatch [:bottom-sheet/show-sheet
-                                      {:content (constantly [requests.actions/request-to-join community])
-                                       :content-height 300}])
-            :override-background-color community-color
-            :style
-            {:width "100%"
-             :margin-top 20
-             :margin-left :auto
-             :margin-right :auto}
-            :before :i/communities}
-           (i18n/label :t/join-open-community)])]
+        (when (and (not joined) can-join?)
+          [:<>
+           [quo/button
+            {:on-press  #(rf/dispatch [:bottom-sheet/show-sheet
+                                       {:content (constantly [requests.actions/request-to-join community])
+                                        :content-height 300}])
+             :override-background-color community-color
+             :style
```

Comment:
This part looks confusing, the other keys have the values in the same line, so could you join the `:style` line with the next one?

```clojure
 :style {:width        "100%"
         :margin-top   20
         :margin-left  :auto
         :margin-right :auto}
```

Reply:
sure will do, I'm really waiting for the auto formatter 😢